mp4/initsegment.go

```diff
@@ -204,7 +204,7 @@ func (t *TrakBox) SetAVCDescriptor(sampleDescriptorType string, spsNALUs, ppsNAL
 }
 
 // SetHEVCDescriptor - Set HEVC SampleDescriptor based on descriptorType and VPS, SPS, and PPS
```

Comment:
This doc string needs an update as well. I missed it in my last review.

Reply:
Ah, I updated it following new docstring strategy as well